hack/user-appstudio-workspace.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/..
+
+if [[ -z ${APPSTUDIO_SP_WORKSPACE} ]]
+then
+  ROOT_WORKSPACE=${ROOT_WORKSPACE:-"root"}
+  APPSTUDIO_WORKSPACE=${APPSTUDIO_WORKSPACE:-"redhat-appstudio"}
+  if [ "${ROOT_WORKSPACE}" == "~" ]; then
+    CURRENT_WS=$(KUBECONFIG=${KCP_KUBECONFIG} kubectl ws . | cut -f2 -d'"')
+    APPSTUDIO_SP_WORKSPACE=${CURRENT_WS}:${APPSTUDIO_WORKSPACE}
+  else
+    APPSTUDIO_SP_WORKSPACE=${ROOT_WORKSPACE}:${APPSTUDIO_WORKSPACE}
+  fi
+fi
+
+USER_APPSTUDIO_WORKSPACE=${USER_APPSTUDIO_WORKSPACE:-"appstudio"}
+echo "Accessing the home workspace:"
+kubectl ws
```

Comment:
good point - I don't like the `'~' ` as you have to use it in apostrophes, otherwise, bash automatically replaces with the home directory path
changed in b48ff3d